cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/service/StreamHandler.java

```diff
@@ -478,7 +478,7 @@ public JsonElement serialize(StreamProperties src, Type typeOfSrc, JsonSerializa
       json.addProperty("name", src.getName());
       json.addProperty("ttl", TimeUnit.MILLISECONDS.toSeconds(src.getTTL()));
       json.add("format", context.serialize(src.getFormat(), FormatSpecification.class));
-      json.addProperty("threshold", src.getThreshold());
+      json.addProperty("notification_threshold_mb", src.getNotificationThresholdMB());
```

Comment:
For the naming, need to double check. Seems like we use `.` to separate words. `notification.threshold`. Also I am not sure if need to carry the unit in the property name (e.g. `ttl` doesn't). Better check with @anew for the naming.